flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatTableHeaderUI.java

```diff
@@ -300,7 +300,7 @@ protected void paintBottomSeparator( Graphics g, JComponent c, int x, int w ) {
 	public Dimension getPreferredSize( JComponent c ) {
 		Dimension size = super.getPreferredSize( c );
 		if( size.height > 0 )
-			size.height = Math.max( size.height, UIScale.scale( height ) );
+			size.height = Math.max( size.height, UIScale.scale( size.height ) );
```

Comment:
This does not look right...
It is actually the same as `size.height = UIScale.scale( size.height )`, isn't it?
But `size.height` is already scaled.
And preferred height (field `height`) is no longer used...